library/testing/TestSDKEngineExceptions.cxx

```diff
@@ -12,8 +12,20 @@ namespace fs = std::filesystem;
 
 int TestSDKEngineExceptions(int argc, char* argv[])
 {
+  // For coverage, check that an invalid symbol loader is not crashing
+  try
+  {
+    std::ignore = f3d::engine::createExternal([](const char*) { return nullptr; });
+  }
+  catch (const f3d::engine::no_window_exception&)
+  {
+  }
```

Comment:
cant use the test.expect framework to implement this one ?

Reply:
not really. No exception is expected to thrown (except if VTK <9.3)